website/source/index.html.erb

```diff
@@ -1,144 +1,434 @@
+        <h2>Hybrid &amp; Multi Cloud</h2>
+        <p class="lead">
+          As more organizations migrate to cloud technologies, the desire to
+          run in multiple datacenters and multiple regions becomes critical.
+          Nomad can span public and private clouds and treat all your
+          infrastructure as a pool of resources.
+        </p>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-12">
+        <%= inline_svg "feature-public-private-clouds.svg" %>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-6">
+        <h3>Hybrid Cloud</h3>
+        <p>
+          Nomad can bridge the gap between the public and private cloud,
+          creating a unified interface for developers to deploy and run any
+          application on any platform. Easily run services across your internal
+          private cloud and external public cloud transparently to application
+          developers.
+        </p>
+      </div>
+      <div class="col-sm-6">
+        <h3>Multi Cloud</h3>
+        <p>
+          As more organizations migrate to cloud technologies, the desire to run
+          applications simultaneously across multiple clouds increases. Nomad
+          combines the power of all cloud providers into a single, unified
+          interface for deploying applications and running services.
+        </p>
       </div>
     </div>
   </div>
-</div>
+</section>
 
-<div id="content">
+<section id="simplify" class="marketing green">
   <div class="container">
     <div class="row">
-      <div class=" col-md-12">
-        <h2 class="feature-section-title">Overview</h2>
+      <div class="col-sm-12">
+        <span class="callout">Maintain</span>
+        <h2>Simplify Operations</h2>
+        <p class="lead">
+          Nomad simplifies operations by supporting blue/green deployments,
+          automatically handling machine failures, and providing a single
+          workflow to deploy applications.
+        </p>
+      </div>
+    </div>
 
-        <div id="deploy" class="feature">
-          <div class="feature-header">
-            <h3>Easily Deploy</h3>
-            <p>
-              Users submit simple high-level jobs and Nomad handles
-              scheduling, deploying and upgrading applications.
-            </p>
-          </div>
-          <div class="feature-graphic"></div>
-          <div class="feature-footer">
-            <p>Nomad makes it easy to deploy one container or thousands.</p>
-            <span class="docker-outline-logo"></span>
-          </div>
-        </div>
+    <div class="row">
+      <div class="col-sm-offset-5 col-sm-6">
+        <h3>Rolling Deploys</h3>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-5" id="feature-rolling-deploys">
+        <%= inline_svg "feature-rolling-deploys.svg" %>
+      </div>
+      <div class="col-sm-7">
+        <p>
+          In order to update an application while reducing downtime, Nomad
```

Comment:
In order to update an application while reducing downtime, Nomad provides a built-in mechanism for rolling upgrades. Operators specify the rate at which they would like to upgrade their service in the declarative job file, plan the changes, and submit the new job. Nomad then updates the service to the newest version using task health and Consul health check information to ensure replacement allocations are healthy. With full control over the update strategy, rolling upgrades make application deployments simple and safe.